lib/awful/util.lua

```diff
@@ -227,29 +227,31 @@ end
 -- @param iconname The name of the icon to search for.
 -- @param exts Table of image extensions allowed, otherwise { 'png', gif' }
 -- @param dirs Table of dirs to search, otherwise { '/usr/share/pixmaps/' }
--- @tparam[opt] string size The size. If this is specified, subdirectories `x`
+-- @tparam[opt] string size The size. If this is specified, subdirectories `sizexsize`
```

Comment:
`sizexsize` is somewhat hard to read and understand if you don't already know the specifics of the directory layout.
Something like this should be easier to understand:

> subdirectories `SIZExSIZE` (e.g. `48x48`)